PantryPal/SecondScreen.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { View, Text, Button, ImageBackground, StyleSheet } from 'react-native';
+
+//Will be the pantry screen
+//Empty right now
+
+const SecondScreen = ({navigation}) => {
+    return (
+        <ImageBackground
+        source={require('./Images/pantryimage.jpg')}
```

Comment:
Cannot use require. It causes the whole app to not function. To resolve, I had to 
import image from './Images/pantryimage.jpg';
and then I had to set source={image} in order to make the app function again.